src/blocks/content-helper/components/post-card.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button, Card, CardBody, CardHeader } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { SuggestedPost } from '../models/suggested-post';
+
+interface PostCardProps {
+	post: SuggestedPost;
+}
+
+function PostCard( { post }: PostCardProps ) {
+	return (
+		<>
+			<Card size="small" elevation={ 2 }>
+				<CardHeader><b>{ post.title }</b></CardHeader>
+				<CardBody>
+					<ul>
+						<li>{ __( 'Views:', 'wp-parsely' ) } { post.views }</li>
+						<li>{ __( 'Published:', 'wp-parsely' ) } { post.date }</li>
+						<li>{ __( 'Author:', 'wp-parsely' ) } { post.author }</li>
+					</ul>
+					<p>
+						<Button href={ post.url } target="_blank" variant="primary">{ __( 'Open Post', 'wp-parsely' ) }</Button> { ' ' }
+						<Button href={ post.statsUrl } target="_blank" variant="secondary">{ __( 'Post Stats', 'wp-parsely' ) }</Button>
+					</p>
+				</CardBody>
+			</Card>
+			<br />
```

Comment:
Is it possible to style this component and avoid literal line breaks?

Reply:
What's the issue we would be trying to avoid?

Reply:
Line breaks are a semantic element and are imprecise at applying vertical spacing. It would be better to provide spacing via CSS.

Reply:
I've tried this but Gutenberg somehow overrides the style and I haven't found a way around this yet. I propose to tackle this in another PR so we can go forward with the release.